### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by cldrn**
A nice place to find the contact info of the team handling security vulnerabilities.

### CODE REVIEW

**Comment by cldrn** [comment]
Who should i contact to report a security vulnerability in private?

**Comment by qqmyers** [comment]
@cldrn - Thanks for the PR - a good idea to make this clearer. If you have an issue to report, security@dataverse.org would be a good contact point.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
+| 5.0.x   | :x:                |
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
```
Comment (pdurbin) [inline SECURITY.md:13]: To make this table more clear for the purpose of my comments below, here's a screenshot:

Screen Shot 2022-01-03 at 10 24 35 AM

I don't know if we write this down anywhere but generally we encourage installations to upgrade in order to receive security updates. If a flaw is found in an old version, we're not very likely to put out a point release to fix it.

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Reporting a Vulnerability
+
+Use this section to tell people how to report a vulnerability.
+
+Tell them where to go, how often they can expect to get an update on a
+reported vulnerability, what to expect if the vulnerability is accepted or
+declined, etc.
+ WHERE TO REPORT SECURITY VULNERABILITIES
```
Comment (pdurbin) [inline SECURITY.md:22]: As @qqmyers already mentioned, security@dataverse.org is the place to report security vulnerabilities. This is mentioned under "Reporting Issues and Contributing" at https://guides.dataverse.org/en/5.8/ as well as CONTRIBUTING.md, the issue template, and https://dataverse.org/contact

This issue is related: "Securing Your Installation" section of Installation Guide could cover ongoing security, advisories, private discussion #3215

**Review by pdurbin — COMMENTED** [review]
Not a bad idea but how common is SECURITY.md? GitHub doesn't seem to recommend it:

Screen Shot 2022-01-03 at 10 33 39 AM

For more on GitHub's recommendations: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/about-community-profiles-for-public-repositories

This issue is related: "Securing Your Installation" section of Installation Guide could cover ongoing security, advisories, private discussion #3215

**Comment by cldrn** [comment]
Hi,
It is suggested on the tab Security between de Wiki and Insights. GH is pushing for projects to use this as it makes it easier to track vulnerabilities and for security researchers to find the right contact. In there you could also explain the expected response time, procedures in general, etc. For example, I contacted security@dataverse.org and they haven't acknowledged the message so I am not sure if that contact was correct at the time.

**Comment by pdurbin** [comment]
@cldrn ah, yes, I see where GitHub suggests SECURITY.md now. Thanks. It looks like you've pretty much used the template.

Also, it looks like your ticket numbers are RT#313094 and RT#313114 but we're just starting to get back from winter break. It'll take some time for a reply. You should have at least received automated emails with the ticket numbers above.

As I mentioned at standup this morning, it probably make sense to have someone from the core team decide what to write in SECURITY.md so maybe one of us should take it from here.

**Comment by djbrooke** [comment]
Hi @cldrn - thanks for this PR. I'm going to close this out and we'll prioritize #3215 to add security.md and to update the Guides with additional security-related information.

**Pull request closed without merging**